index.html

```diff
@@ -52,14 +52,14 @@ <h2 id="main-h2">Private Digital Currency</h2>
                         </div>
                         <div class="col-lg-6 col-md-6 col-sm-8 col-xs-12 why-text">
                             <h3>Monero is private.</h3>
-                            <p>The power of the blockchain usually increases security at the cost of privacy, but with Monero's sophisticated privacy-centric technology, you get all of the security benefits of the blockchain without any of the privacy trade-offs.</p>
+                            <p>Security of the blockchain normally increases at the cost of privacy. Monero is based on the Cryptonote application layer protocol, which allows for the security benefits of blockchain technology without any of the privacy trade-offs</p>
                         </div>
                         
                     </div>
                     <div class="row middle-xs info-block-row">
                         <div class="col-lg-6 col-md-6 col-sm-8 col-xs-12 why-text">
                             <h3>Monero is untraceable.</h3>
-                            <p>By taking advantage of ring signatures, Monero makes it ambiguous which funds have been spent, and thus extremely unlikely that a transaction could be linked to particular user.</p>
+                            <p>Sending and receiving addresses as well as transacted amounts are obfuscated. Transactions cannot be linked to a particular user or real-world identity.</p>
```

Comment:
> Maybe 'obfuscated by default'?

I think `private by default` has become somewhat of a tagline for Monero. I think we should use it as much as possible if appropriate.

>Sending and receiving addresses as well as transacted amounts are obfuscated.

I think the word `hidden' is better in this context.

For SEO I think it might be good if we keep the word `Monero` in there somewhere. Maybe something like:
`With Monero, sending and recieving addresses are hidden and amounts are obfuscated. With ring-signatures, Monero makes it's ambiguous who is sending funds, and thus it's extremely unlikely that a transaction could be linked to a particular user or real-world identity.`